### ISSUE TITLE
Unused import/function in the recent release

### ISSUE THREAD

**Issue by orhun** [Hyde46/hoard#281]
Hello! 🐻

While updating the AUR package for `hoard`, I realized there are a couple of unused things:

```
warning: unused import: `DrawState`
 --> src/gui/list_search/key_not_set_controls.rs:2:46
  |
2 | use crate::gui::commands_gui::{ControlState, DrawState, State};
  |                                              ^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

warning: unused import: `chatgpt::prelude::ChatGPT`
  --> src/hoard.rs:27:5
   |
27 | use chatgpt::prelude::ChatGPT;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^

warning: associated function `from_gpt_string` is never used
   --> src/command/hoard_command.rs:244:12
    |
18  | impl HoardCommand {
    | ----------------- associated function in this implementation
...
244 |     pub fn from_gpt_string(gpt_string: &str) -> Self {
    |            ^^^^^^^^^^^^^^^
    |
    = note: `#[warn(dead_code)]` on by default

warning: `hoard-rs` (bin "hoard") generated 3 warnings (run `cargo fix --bin "hoard"` to apply 2 suggestions)
```

Just wanted to ask if they are a part of an incomplete implementation or left there on purpose. I might submit a PR to remove the warnings if needed.

Thank you!

**Comment by Hyde46** [comment]
Thanks for the issue!
I hastily put the release together which i shouldn't have, i'll fix it in the coming days 😃

**Comment by Hyde46** [comment]
Fixed in #282

**Comment by orhun** [comment]
Thanks!
